### ISSUE TITLE
generated shell script fails to run inside docker container because of empty line before shebang unix

### ISSUE THREAD

**Issue by jm2dev** [xerial/sbt-pack#119]
Generated shell script under target/pack/bin looks like:

```

#!/bin/bash
# bla bla
```

though it runs on my linux machine, it fails to run inside docker container. 
The fix consists on removing the first empty line, but ideally it shouldn't appear on generation.

Desired output:

```
#!/bin/bash
# bla bla
```

**Comment by woky** [comment]
Create very simple systemd service file in `/etc/systemd/system/foo.service` to start the program by the launch script (`start`) created by pack:

```
[Service]
ExecStart=/opt/foo/bin/start
```

Start the service: `systemctl start foo`

The service won't start and the following will appear in journal (`journalctl -n20`):
```
Nov 05 20:50:26 myhost systemd[291]: foo.service: Failed to execute command: Exec format error
Nov 05 20:50:26 myhost systemd[291]: foo.service: Failed at step EXEC spawning /opt/foo/bin/start: Exec format error
```

The problem is that the first line of the launch script is empty line. I think systemd is being correct (and perhaps strict) here. The shebang line should be the first line in the file.

Perhaps the fix could be something like this?

```patch
diff --git a/src/main/twirl/xerial/sbt/pack/launch.scala.txt b/src/main/twirl/xerial/sbt/pack/launch.scala.txt
index 632d466..1a13ffd 100644
--- a/src/main/twirl/xerial/sbt/pack/launch.scala.txt
+++ b/src/main/twirl/xerial/sbt/pack/launch.scala.txt
@@ -1,5 +1,4 @@
-@(opts:xerial.sbt.pack.LaunchScript.Opts, expandedClasspath:Option[String])
-#!/bin/sh
+@(opts:xerial.sbt.pack.LaunchScript.Opts, expandedClasspath:Option[String])#!/bin/sh
 #/*--------------------------------------------------------------------------
 # *  Copyright 2012 Taro L. Saito
 # *
```

**Comment by gijsbert802** [comment]
I just ran into this issue. For now I worked around it by using `CMD ["sh", "bin/pack-generated-script.sh"]` instead of just `CMD["bin/pack-generated-script.sh"]` in my Dockerfile.

**Comment by xerial** [comment]
Try sbt-0.9.2 with a fix in #120

**Comment by Paulpeter** [comment]
Was struggling with this problem for some time where the script run fine but failed to execute with service, putting the shebang in the first line solved it
